content/en/infrastructure/containers/container_images.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Container Images
+kind: documentation
+further_reading:
+- link: "/security/vulnerabilities"
+  tag: "Documentation"
+  text: "Cloud Security Management Vulnerabilities"
+- link: "/security/vulnerabilities/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting Cloud Security Management Vulnerabilities"
+- link: "/security/cloud_security_management/setup/csm_pro?tab=aws#configure-the-agent-for-hosts-and-containers"
+  tag: "Documentation"
+  text: "Setting up CSM container vulnerabilities"
+- link: "/security/cloud_security_management/setup/csm_enterprise?tab=aws#configure-the-agent-for-vulnerabilities"
+  tag: "Documentation"
+  text: "Setting up CSM host vulnerabilities"
+- link: "https://www.datadoghq.com/blog/datadog-container-image-view/"
+  tag: "Blog"
+  text: "Enhance your troubleshooting workflow with Container Images in Datadog Container Monitoring"
+---
+
+## Overview
+
+The [Container Images][1] view in Datadog provides key insights into every image used in your environment, helping you detect and remediate security and performance issues that can affect multiple containers. You can view container image details alongside the rest of your container data, allowing you to troubleshoot image issues affecting infrastructure health. Additionally, you can view vulnerabilities found in your container images from [Cloud Security Management][2] (CSM) to help you streamline your security efforts.
```

Comment:
It wold be interesting to mention the ability to quickly assess deployment footprint using the new page.
Maye something like
```
The [Container Images][1] view in Datadog provides key insights into every image used in your environment, helping you assess their deployment footprint, detect and remediate security and performance issues that can affect multiple containers.
```